### PR TITLE
Create wiki

### PR DESCRIPTION
**Pull request by manavluhar**
Adding wiki for  **Adobe Commerce Support Knowledge Base**

## Purpose of this pull request
 
This pull request (PR) ...
 
## Affected Support KB pages
 

 
List the affected pages on support.magento.com (URLs).

### CODE REVIEW

File: wiki [hunk 1]
```diff
@@ -0,0 +1,6 @@
+## Welcome to the Adobe Commerce Support Knowledge Base!
+This repository contains the markdown source of articles, published in Adobe Commerce Support Knowledge Base.
```
Comment (BarnyShergold) [inline wiki:2]:
```suggestion
This repository contains the markdown source of articles, published in the Adobe Commerce Support Knowledge Base.
```

File: wiki [hunk 2]
```diff
@@ -0,0 +1,6 @@
...
+
+### Become a contributor
+Getting started with our contribution program is easy. You can start with a simple page edit, and then work your way up to major revisions and new topics. The more participation we have from the Magento community, the better the docs are for everyone!
```
Comment (BarnyShergold) [inline wiki:5]:
```suggestion
Getting started with our contribution program is easy. You can start with a simple page edit, then work your way up to major revisions and new topics. The more participation we have from the Magento community, the better the docs are for everyone!
```

**Review by BarnyShergold — CHANGES_REQUESTED** [review]

**Comment by manavluhar** [comment]
Thank you @BarnyShergold I accepted suggestions.

**Comment by almarchenko** [comment]
@magento import code to magento-kb/knowledge-base

**Comment by magento-engcom-team** [comment]
@almarchenko the branch with code successfully imported into`magento-kb/knowledge-base` repository. Branch name: `imported-magento-knowledge-base-46`.

**Review by almarchenko — APPROVED** [review]

**Review by BarnyShergold — APPROVED** [review]

**Comment by almarchenko** [comment]
@manavluhar  I'm not sure publishing a wiki page is possible using our existing process, but I will try! If it doesn't get through, we'll add manually and add points manually. (unless I find some better solution)

**Comment by almarchenko** [comment]
@magento import code to magento-kb/knowledge-base

**Comment by magento-engcom-team** [comment]
@almarchenko the branch with code successfully imported into`magento-kb/knowledge-base` repository. Branch name: `imported-magento-knowledge-base-46`.

**Pull request merged**